semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/CfgNodeMethods.scala

```diff
@@ -94,6 +94,26 @@ class CfgNodeMethods(val node: CfgNode) extends AnyVal with NodeExtension {
     }
   }
 
+  /** Using the post dominator tree, will determine if this node passes through the included set of nodes and filter it
+    * in.
+    * @param included
+    *   the nodes this node must pass through.
+    * @return
+    *   the traversal of this node if it passes through the included set.
+    */
+  def passes(included: Set[CfgNode]): Traversal[CfgNode] =
+    node.filter(_.postDominatedBy.toSet.exists(included.contains))
+
+  /** Using the post dominator tree, will determine if this node passes through the excluded set of nodes and filter it
+    * out.
+    * @param excluded
+    *   the nodes this node must not pass through.
+    * @return
+    *   the traversal of this node if it does not pass through the excluded set.
+    */
+  def passesNot(excluded: Set[CfgNode]): Traversal[CfgNode] =
+    node.filterNot(_.postDominatedBy.toSet.exists(excluded.contains))
```

Comment:
```suggestion
    node.filterNot(_.postDominatedBy.exists(excluded.contains))
```
no need to convert the Traversal to a Set afaics (?)